src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -338,28 +338,56 @@ The following are an example `config.toml` files:
    url="http://127.0.0.1:50051"
 
    [[handlers]]
-   cosmwasm_contract="[TBD]"
+   cosmwasm_contract="axelar14a4ar5jh7ue4wg28jwsspf23r8k68j7g5d6d3fsttrhp42ajn4xq6zayy5"
    type="MultisigSigner"
 
-   # For each supported chain
    [[handlers]]
-   chain_name="[chain name]"
-   chain_rpc_url="[node rpc url]"
-   cosmwasm_contract="[voting verifier address]"
-   chain_finalization="[RPCFinalizedBlock or ConfirmationHeight]"
+   chain_name="test-sepolia"
+   chain_rpc_url="https://rpc.ankr.com/eth_sepolia"
+   cosmwasm_contract="axelar1r4rmvn83vrfj4evy5l8cv2nat2v0whm36ds3crn2mhlq8ufmhvts9467zz"
+   chain_finalization="RPCFinalizedBlock
    type="EvmMsgVerifier"
 
    [[handlers]]
-   chain_name="[chain name]"
-   chain_rpc_url="[node rpc url]"
-   cosmwasm_contract="[voting verifier address]"
-   chain_finalization=["RPCFinalizedBlock" or "ConfirmationHeight"]
+   chain_name="test-sepolia"
+   chain_rpc_url="https://rpc.ankr.com/eth_sepolia"
+   cosmwasm_contract="axelar1r4rmvn83vrfj4evy5l8cv2nat2v0whm36ds3crn2mhlq8ufmhvts9467zz"
+   chain_finalization=RPCFinalizedBlock"
    type="EvmVerifierSetVerifier"
+
+   [[handlers]]
+   chain_name="test-avalanche"
+   chain_rpc_url="https://api.avax-test.network/ext/bc/C/rpc"
+   cosmwasm_contract="axelar1hupk5du59cgu4ps5s637rhakwsq0060ycdp57j2ccevna7wqqzrqnfrr0p"
+   chain_finalization="ConfirmationHeight"
+   type="EvmMsgVerifier"
+
+   [[handlers]]
+   chain_name="test-avalanche"
+   chain_rpc_url="https://api.avax-test.network/ext/bc/C/rpc"
+   cosmwasm_contract="axelar1hupk5du59cgu4ps5s637rhakwsq0060ycdp57j2ccevna7wqqzrqnfrr0p"
+   chain_finalization="ConfirmationHeight"
+   type="EvmVerifierSetVerifier"
+
+   # For each supported chain
+   #[[handlers]]
```

Comment:
can we remove this commented code?

Reply:
This isn't commented code, this is the template verifiers should  use for each chain they want to add.